src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -175,6 +200,10 @@ end
 
 -- seals the named token on this card
 function sealToken(name, playerColor)
+  if #sealedTokens == MAX_SEALED then
```

Comment:
Should probably be a `>=` for good measure? Not sure if there is a way that there could be more tokens sealed, but might as well cover that